registration/include/pcl/registration/correspondence_rejection.h

```diff
@@ -198,7 +198,7 @@ class CorrespondenceRejector {
 
 protected:
   /** \brief The name of the rejection method. */
-  std::string rejection_name_;
+  std::string rejection_name_{"CorrespondenceRejectorPoly"};
```

Comment:
How did this happen?

Reply:
Got it from the constructor body, I put it into the constructor initialization list instead.